cmd/influxd/run/server.go

```diff
@@ -487,6 +487,16 @@ func (s *Server) Open() error {
 		// so it only logs as is appropriate.
 		s.QueryExecutor.TaskManager.Logger = s.Logger
 	}
+	if s.config.Data.QueryLogPath != "" {
+		path := s.config.Data.QueryLogPath
+		flw := query.NewFileLogWatcher(s.QueryExecutor, path, s.Logger, s.config.Logging.Format)
```

Comment:
Either `query.NewFileLogWatcher` is misnamed, or `QueryExecutor.WithLogWriter` is misnamed. It looks like the wrong thing is getting passed around.